### PR TITLE
bump up the version to 3.2.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by gsps1**

### CODE REVIEW

**Comment by gokulavasan** [comment]
LGTM :+1:

**Comment by anwar6953** [comment]
Should you first merge the following branch into develop:
https://github.com/caskdata/cdap-apps/compare/release/cdap-3.1-compatible

**Comment by gsps1** [comment]
yes @anwar6953 PR is here : https://github.com/caskdata/cdap-apps/pull/91

File: Wise/pom.xml [hunk 1]
```diff
@@ -16,8 +16,8 @@
     <hadoop.version>[2.0.2-alpha, 2.3.0]</hadoop.version>
     <surefire.redirectTestOutputToFile>true</surefire.redirectTestOutputToFile>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <co.cask.cdap.version>3.1.0</co.cask.cdap.version>
-  </properties>
+    <co.cask.cdap.version>3.2.0-SNAPSHOT</co.cask.cdap.version>
```
Comment (anwar6953) [inline Wise/pom.xml]: for consistency, please change this property name to `cdap.version`.
Otherwise, LGTM.

Comment (anwar6953) [inline Wise/pom.xml]: also, looks like you un-indented the `</properties>`

Comment (gsps1) [inline Wise/pom.xml]: updated. the </properties> comes in the next line now. just the diff..

File: Wise/pom.xml [hunk 2]
```diff
@@ -16,8 +16,8 @@
     <hadoop.version>[2.0.2-alpha, 2.3.0]</hadoop.version>
     <surefire.redirectTestOutputToFile>true</surefire.redirectTestOutputToFile>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <co.cask.cdap.version>3.1.0</co.cask.cdap.version>
-  </properties>
+    <cdap.version>3.2.0-SNAPSHOT</cdap.version>
+ </properties>
```
Comment (anwar6953) [inline Wise/pom.xml]: why de-dent this, though?

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**